@xen-orchestra/lite/src/pages/settings.vue

```diff
@@ -1,134 +1,128 @@
-                  {{ t('community-name', { name: 'XCP-ng' }) }}
-                </a>
-              </template>
-            </UiKeyValueRow>
-            <UiKeyValueRow>
-              <template #key>{{ t('documentation') }}</template>
-              <template #value>
-                <a target="_blank" rel="noopener noreferrer" href="https://xcp-ng.org/docs/">
-                  {{ t('documentation-name', { name: 'XCP-ng' }) }}
-                </a>
-              </template>
-            </UiKeyValueRow>
-            <UiKeyValueRow>
-              <template #key>{{ t('support') }}</template>
-              <template #value>
-                <a target="_blank" rel="noopener noreferrer" href="https://xcp-ng.com/">
-                  {{ t('professional-support') }}
-                </a>
-              </template>
-            </UiKeyValueRow>
-          </UiKeyValueList>
-        </div>
+  <UiCard class="settings-page">
+    <UiTitle>{{ t('about') }}</UiTitle>
+    <VtsColumns class="about-columns">
+      <VtsQuickInfoColumn>
+        <h5>Xen Orchestra Lite</h5>
+        <VtsQuickInfoColumn>
+          <VtsQuickInfoRow :label="t('version')">
+            <template #value>
+              {{ `v${xoLiteVersion}` }}
+              <code v-if="xoLiteGitHead">{{ `(${xoLiteGitHead.slice(0, 5)})` }}</code>
+            </template>
+          </VtsQuickInfoRow>
+          <VtsQuickInfoRow :label="t('news')">
+            <template #value>
+              <UiLink size="medium" href="https://xen-orchestra.com/blog/">
+                {{ t('news-name', { name: 'Xen Orchestra' }) }}
+              </UiLink>
+            </template>
+          </VtsQuickInfoRow>
+          <VtsQuickInfoRow :label="t('community')">
+            <template #value>
+              <UiLink size="medium" href="https://xcp-ng.org/forum/category/12/xen-orchestra">
+                {{ t('community-name', { name: 'Xen Orchestra' }) }}
+              </UiLink>
+            </template>
+          </VtsQuickInfoRow>
+          <!-- #TODO we dont have ofical documentation for xo-lite -->
+          <!--
+ <VtsQuickInfoRow :label="t('documentation')">
+            <template #value>
+              <UiLink size="medium" href="https://github.com/vatesfr/xen-orchestra/tree/master/%40xen-orchestra/lite">
+                {{ t('documentation-name', { name: 'XO Lite' }) }}
+              </UiLink>
+            </template>
+          </VtsQuickInfoRow>
+-->
+        </VtsQuickInfoColumn>
+      </VtsQuickInfoColumn>
+      <VtsQuickInfoColumn>
+        <h5>XCP-ng</h5>
+        <VtsQuickInfoColumn>
+          <VtsQuickInfoRow :label="t('version')">
+            <template #value>{{ `v${xcpVersion}` }}</template>
+          </VtsQuickInfoRow>
+          <VtsQuickInfoRow :label="t('news')">
+            <template #value>
+              <UiLink size="medium" href="https://xcp-ng.org/blog/">
+                {{ t('news-name', { name: 'XCP-ng' }) }}
```

Comment:
```suggestion
                {{ t('news-name', { name: t('xcp-ng') }) }}
```
Maybe you can use your key translate ( for xen orchestra too)